.github/workflows/build-release.yml

```diff
@@ -39,6 +42,11 @@ jobs:
           - 'linux-arm64v8'
           - 'win32-ia32'
           - 'win32-x64'
+        experimental: [false]
+        include:
+          - os: macos-latest
```

Comment:
Given there are no containers involved that we can use to "pin" the versions of any OS shared library dependencies, should we pin to a specific macOS release e.g. `macos-10.15`?

Reply:
Yes, lets pin it to `macos-10.15`. Commit https://github.com/lovell/sharp-libvips/pull/44/commits/3dbab0f89b50f3126bc43928663e782847fdfae8.